source/_components/ihc.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: "IHC"
+description: "Instructions on how to integrate the IHC components with Home Assistant"
+date: 2017-11-11 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ihc.png
+ha_category: Hub
+ha_release: "0.59"
+ha_iot_class: "Local Push"
+---
+
+IHC Controller integration for Home Assistant allows you to connect the LK IHC controller to Home Assistant. 
+(The controller is sold under other names in different countries - "ELKO Living system" in Sweeden and Norway)
+
+An `ihc` section must be present in the `configuration.yaml` file and contain the following options:
+
+```yaml
+# Example configuration.yaml entry
+ihc:
+   host: http://192.168.1.3
+   username: admin
+   password: mysecret
+   auto_setup: True
+   info: True
+```
+Configuration variables:
+
+- **auto_setup** (*Optional*): True to have IHC products auto setup.
+- **host** (*Required*): The url of the IHC Controller.
+- **info** (*Optional*): If True additional ihc info will be shown on each component.
+- **password** (*Required*): The password for the IHC Controller.
+- **username** (*Required*): The username for the IHC Controller.
+
+The info option will show the IHC "name", "note" and "position" attributes. 
+This will make it easier to identify the IHC products within Home Assistant
+
+There is currently support for the following device types within Home Assistant:
+
+- [Binary Sensor](../binary_sensor.ihc/) 
+- [Sensor](../sensor.ihc/) 
+- [Light](../light.ihc/) 
+- [Switch](../switch.ihc/) 
+
+### Auto setup of IHC products
+
+If auto setup is enabled, the ihc component will automatically find IHC products and insert these as devices in Home Assistant.
```

Comment:
- Place backticks around the `ich` when referencing to the component.

Reply:
Done